felix/dataplane/driver.go

```diff
@@ -394,7 +394,6 @@ func StartDataplaneDriver(
 			RouteTableManager:                  routeTableIndexAllocator,
 			MTUIfacePattern:                    configParams.MTUIfacePattern,
 			BPFExcludeCIDRsFromNAT:             configParams.BPFExcludeCIDRsFromNAT,
-			NfNetlinkBufSize:                   configParams.NfNetlinkBufSize,
```

Comment:
This makes sense, but will also create conflict on merge ... :man_shrugging:

Reply:
I think it make sense to have all fields related to flow logs in one place. The conflict will happen once, and can be managed easily.